packages/genesys-spark-components/src/components/stable/gux-pagination-cursor/gux-pagination-cursor.tsx

```diff
@@ -42,6 +46,12 @@ export class GuxPaginationCursor {
   @Prop()
   layout: 'simple' | 'advanced' = 'simple';
 
+  /**
+   * ID of the element that contains the paginated content to aid accessibility.
+   */
+  @Prop()
+  controls: string;
```

Comment:
I don't think it is possible to pass ids into the shadowdom and have them link elements in the light and shadow dom.